projects/storefrontlib/src/lib/storefront.module.ts

```diff
@@ -36,7 +37,8 @@ import { StoreFinderModule } from './store-finder/store-finder.module';
     CxApiModule,
     SmartEditModule.forRoot(),
     MultiStepCheckoutModule,
-    I18nModule.forRoot()
+    I18nModule.forRoot(),
```

Comment:
what happened to the trailing comma?

Reply:
The changes are not merged yet in develop I believe.

Reply:
Strange, I branched off of develop

Reply:
The changes are _*not_* merged yet in develop I believe, so the trailing commas aren't there yet.

Reply:
gotcha :)